api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -155,7 +162,7 @@ type AzureManagedControlPlaneSpec struct {
 
 	// LoadBalancerSKU is the SKU of the loadBalancer to be provisioned.
 	// Immutable.
-	// +kubebuilder:validation:Enum=Basic;Standard
+	// +kubebuilder:validation:Enum=Basic;basic;Standard;standard
```

Comment:
what's the reason for supporting both? Would it be easier to keep supporting one and do the conversion in CAPZ?

Reply:
Yeah, supporting only one set is likely easier. Just pushed that change.